pkg/diff/diff.go

```diff
@@ -865,8 +866,8 @@ func (d *differ) diffTarEntries(ctx context.Context, node *EventTreeNode, in [2]
 
 func (d *differ) diffTarEntry(ctx context.Context, node *EventTreeNode, in [2]EventInput) (dirsToBeRemovedIfEmpty []string, retErr error) {
 	var negligibleTarFields []string
-	if d.o.IgnoreTimestamps {
```

Comment:
This seems to break the current behavior of `IgnoreTimestamps`

Reply:
Timestamps on the docker metadata and timestamps in the filesystem are two different beasts and in my specific use-case having a single switch is insufficient.

When I rebuild from cache I want to make sure that the filesystem is identical, but a new docker image will be emitted and therefore updating the timestamp on the image metadata.

Reply:
yes it's a breaking interface change, alternative would be to expose two new flags:
```
ignore-file-timestamps
ignore-image-timestamps
```

and bundle them together like we do with `semantic`